.dis-vulncheck.yaml

```diff
@@ -1,2 +1,2 @@
 ---
-toolchain: go1.24.9
+toolchain: go1.24.11
```

Comment:
This is fine, but probably won't be needed soon when [this dis-vulncheck PR](https://github.com/ONSdigital/dis-vulncheck/pull/4) is merged